apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsCollectionDropdownControl.tsx

```diff
@@ -0,0 +1,70 @@
+import type { ControlProps, RankedTester } from "@jsonforms/core"
+import { FormControl, Skeleton } from "@chakra-ui/react"
+import { and, rankWith, schemaMatches } from "@jsonforms/core"
+import { withJsonFormsControlProps } from "@jsonforms/react"
+import { FormLabel, SingleSelect } from "@opengovsg/design-system-react"
+import { COLLECTION_DROPDOWN_FORMAT } from "@opengovsg/isomer-components"
+
+import Suspense from "~/components/Suspense"
+import { JSON_FORMS_RANKING } from "~/constants/formBuilder"
+import { siteSchema } from "~/features/editing-experience/schema"
+import { useQueryParse } from "~/hooks/useQueryParse"
+import { trpc } from "~/utils/trpc"
+
+export const jsonFormsCollectionDropdownControlTester: RankedTester = rankWith(
+  JSON_FORMS_RANKING.RefControl,
```

Comment:
issue: should use a different ranking name

Reply:
Eh haha bugbot is right, I meant that each tester ranking should be its own name, like here should be called `JSON_FORMS_RANKING.DropdownControl` in case we need to change it to a different value for some reason in the future

Reply:
thought about it but felt like they are different UI variations of multi-select (AnyOf)

the only difference is
1. which queries they call e.g. getCollections, getCategories, and 
2. how to display them (dropdown, radio etc.)

As such, don't think there's a need to differentiate them as it wouldn't and shouldn't affect the ranking order though

Reply:
Hmm I think you might be reading too much into the comment. It's more of each form control should have its own ranking name so it's easier to see the 1-to-1 mapping. The actual numbers for the ranking sure, they can be the same number.

If we ever need to adjust the ranking integer for the dropdown control only, it's easy to switch, not that we would ever need to but just in case.